src/context_store/storage/prisma.py

```
@@ -423,26 +587,33 @@ async def _ensure_system_migration(self) -> None:
+            logger.exception("Failed to fetch applied migrations from schema_migrations table")
             raise
 
     async def _tables_exist(self, tables: list[str]) -> bool:
+        """Check if specified tables exist in the database."""
         sql = "SELECT tablename FROM pg_catalog.pg_tables WHERE tablename = ANY($1)"
-        rows = await self._client.query_raw(sql, tables)
-        return len(rows) >= len(tables)
+        try:
+            rows = await self._client.query_raw(sql, tables)
+            return len(rows) >= len(tables)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)